nativescript-angular/renderer.ts

```diff
@@ -46,6 +47,9 @@ export class NativeScriptRootRenderer implements RootRenderer {
 
     public get rootView(): View {
         if (!this._rootView) {
+            this._rootView = NativeScriptPlatformRef.rootPage;
```

Comment:
if (!this._rootView) {
    this._rootView = NativeScriptPlatformRef.rootPage || topmost().currentPage;
}
instead?